charts/osm/values.yaml

```diff
@@ -330,9 +444,19 @@ osm:
     # -- Log level for the multicluster gateway
     gatewayLogLevel: error
 
-  # -- Node tolerations applied to control plane pods.
-  # The specified tolerations allow pods to schedule onto nodes with matching taints.
-  controlPlaneTolerations: []
+    ## Node labels for pod assignment
+    ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+    nodeSelector:
+      kubernetes.io/arch: amd64
```

Comment:
@shashankram The envoy-alpine image is only built for amd64. Do you see any issues if we specify `envoyproxy/envoy` as the default sidecar image instead which also supports arm64?